translations/strings.yaml

```diff
@@ -244,9 +244,18 @@ customDNSSettings:
 networkSettings:
   multihopTitle: Multi-hop tunnel
 
+
+captivePortalAlert:
+  title: Captive portal detected!
+  header: A captive portal is when the guest Wi-Fi you are using requires you to log in to get internet access. 
+  actionPreActivation: You can turn on your VPN once you have gained internet access by logging in to the portal. 
+  actionPostActivation: You must turn off your VPN in order to see the portal, and turn it back on once you have logged in. 
+  buttonTextPreActivation: Okay
```

Comment:
Note: I've seen complains from people about this form vs OK (at least one user on Bugzilla). Not a localization problem, just worth thinking about it ;-)